internal/pkg/client/oci/nativesif.go

```diff
@@ -75,20 +73,20 @@ func convertOciToSIF(ctx context.Context, imgCache *cache.Handle, image, cachedI
 	b, err := build.NewBuild(
 		image,
 		build.Config{
-			Dest:      cachedImgPath,
-			Format:    "sif",
-			NoCleanUp: opts.NoCleanUp,
+			Dest:   cachedImgPath,
+			Format: "sif",
 			Opts: buildtypes.Options{
 				TmpDir:           opts.TmpDir,
 				NoCache:          imgCache.IsDisabled(),
 				NoTest:           true,
 				NoHTTPS:          opts.NoHTTPS,
-				DockerAuthConfig: opts.OciAuth,
 				DockerDaemonHost: opts.DockerHost,
 				ImgCache:         imgCache,
 				Platform:         opts.Platform,
+				OCIAuthConfig:    opts.OciAuth,
```

Comment:
If we've renamed this from `DockerAuthXYZ` to `OCIAuthXYZ`, why not rename `DockerAuthFile` (next line) to `OCIAuthFile` as well?

Reply:
The auth file is _not_ an OCI standard. The format of this file is specifically something that is defined by Docker's implementation, therefore it's a DockerAuthFile.

Reply:
To be fair - the AuthConfig isn't an OCI standard either... but is is something that applies to accessing OCI registries of all kinds.